### PR TITLE
Fixed Name & URI 1 Fields Populated Incorrectly

### PR DESCRIPTION
**Pull request by arunkapur**
Fixed Name & URI 1 Fields populated incorrectly when creating new login in popout window (#1816)

### CODE REVIEW

File: src/popup/vault/add-edit.component.ts [hunk 1]
```diff
@@ -76,10 +76,21 @@ export class AddEditComponent extends BaseAddEditComponent {
 
             if (!this.editMode || this.cloneMode) {
                 if (params.name && (this.cipher.name == null || this.cipher.name === '')) {
-                    this.cipher.name = params.name;
+                    if (this.popupUtilsService.inPopout(window)) {
+                        this.cipher.name = '';
```
Comment (eliykat) [inline src/popup/vault/add-edit.component.ts:80]: Do we need to initialize it with an empty string in this case, or could we just skip the assignment altogether? For example - you could check for `&& !this.popupUtilsService.inPopout(window)` in the existing `if` statement on line 78. (and same below for the URI.) I haven't tested this though, there could be a reason for it.

**Review by eliykat — COMMENTED** [review]
Thanks @arunkapur! Looking good, just 1 question about whether the empty string assignment is necessary.

File: src/popup/vault/add-edit.component.ts (hunk 1 above)
Comment (arunkapur) [reply src/popup/vault/add-edit.component.ts]: That works as well. Thanks for the suggestion. I pushed those changes.

**Review by eliykat — APPROVED** [review]
Thanks @arunkapur !

**Pull request merged**